internal/globpath/globpath.go

```diff
@@ -21,7 +21,7 @@ func Compile(path string) (*GlobPath, error) {
 	out := GlobPath{
 		hasMeta:      hasMeta(path),
 		HasSuperMeta: hasSuperMeta(path),
-		path:         filepath.FromSlash(path),
+		path:         path,
```

Comment:
Will this change the style of the paths returned?  If so this could be a breaking change and we need to be very careful.  Please double and triple check the changes in this file.

Reply:
With fixing other test I had to make additional changes. All the fixed lead to the state where paths are always compared in the same form (forward slash as a path separator). It will not change any path returned